tests/react/link.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('Link', () => {
     expect(routerContext.push).not.toHaveBeenCalled()
     container.querySelector('a')?.click()
     expect(routerContext.push).toHaveBeenCalledWith(
-      '/community/[communityId]/[communitySlug]/statistics?communityId=300&communitySlug=three-hundred&baz=3',
+      '/communaute/[communityId]/[communitySlug]/statistiques?communityId=300&communitySlug=three-hundred&baz=3',
```

Comment:
It should be `'/community/[communityId]/[communitySlug]/statistics?communityId=300&communitySlug=three-hundred&baz=3'` - a file url - so that Next understands it.

Reply:
I think Next completely understands that url otherwise it would not be working on the demo that I linked to this PR and that is the desire behaviour.

I will definitely expect that specific path to be translated to french as you're passing `fr` as locale.

```js
{ locale: 'fr', scroll: undefined, shallow: undefined },

```

If the returned url is in english all the problems about the links not taking you were they should and making you refresh the page in order to them to redirect properly will continue happening.